### PR TITLE
US16

### PR DESCRIPTION
**Pull request by Unikorn44**
Replacing US15, bugs fixed for merge

### CODE REVIEW

**Comment by Unikorn44** [comment]
Code corrected for merge

File: src/Model/ConnectManager.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+     * Update item in database
+     */
+    public function update(array $item): bool
+    {
+        $statement = $this->pdo->prepare("UPDATE " . self::TABLE . " SET `title` = :title WHERE id=:id");
+        $statement->bindValue('id', $item['id'], \PDO::PARAM_INT);
+        $statement->bindValue('title', $item['title'], \PDO::PARAM_STR);
+
+        return $statement->execute();
+    }
```
Comment (aldaitzwild) [inline src/Model/ConnectManager.php:31]: No ! What is this code ? It is not used anywhere. It doesn't work if you use it. So why do you make me read this ?

File: src/View/Home/index.html.twig [hunk 2]
```diff
@@ -1,5 +1,107 @@
...
+            height: 25px;
+        }*/
+        /*#close_popup img::selection{ /*peut être supprimé*/
+           /* display: none;
+        }*/
+        /*#close_popup:hover, #close_popup:focus{
+            color: #000;
+            text-decoration: none;
+            cursor: pointer;
+        }*/
```
Comment (aldaitzwild) [inline src/View/Home/index.html.twig:52]: No Dead code ! 
You should never commit commented code

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/Home/index.html.twig (hunk 2 above)
Comment (Unikorn44) [reply src/View/Home/index.html.twig]: dead code suppressed
Sorry for the useless code

**Review by aldaitzwild — APPROVED** [review]
Yes ! We're good now !

**Pull request merged**